packages/nextra-theme-docs/src/constants.tsx

```diff
@@ -301,6 +306,25 @@ export const DEFAULT_THEME: DocsThemeConfig = {
     titleComponent: ({ title }) => <>{title}</>,
     toggleButton: false
   },
+  themeSwitch: {
+    component: ThemeSwitch,
+    options: function useOptions() {
```

Comment:
```suggestion
    useOptions: function useOptions() {
```

I remember @shuding prefer `use` prefix for config options where you can use react hooks

Reply:
still I think it's better to call it `useLabels`, or if you want to keep `useOptions` then, return `{labelLight, labelDark, labelSystem}`

Reply:
> still I think it's better to call it `useLabels`, or if you want to keep `useOptions` then, return `{labelLight, labelDark, labelSystem}`

Bad idea, I don't think calling items in a select as "label" is suitable...

Reply:
The `options` property has been renamed to `useOptions`! Also improved the documentation as well